lib/ansible/modules/cloud/azure/azure_rm_webapp.py

```diff
@@ -410,6 +420,11 @@ def __init__(self):
                 type='bool',
                 default=False
             ),
+            app_state=dict(
+                type='str',
+                choices=['start', 'stop', 'restart'],
```

Comment:
hehe, still wondering, would be better to have "started", "stopped", "restarted"?

Reply:
or, "running", "stopped", "restart"?